data/recData/notify.js

```diff
@@ -0,0 +1,69 @@
+class Notify {
+  constructor() {
+    self.port.on('data', recs => {
+      this.removeOldBoxes();
+      this.createNewBoxes(recs);
+    });
+  }
+
+  //removes old recommendations, that may be for a different domain
+  removeOldBoxes() {
+    var recDiv = document.getElementById('recs');
+    while(recDiv.firstChild) {
+      recDiv.removeChild(recDiv.firstChild);
+    }
+  }
+
+
+  createNewBoxes(recs) {
+    for(var i = 0; i < recs.length; i++) {
+      var box = this.createNewBox();
+      this.fillInValues(box, recs[i]);
+    }
+  }
+
+
+  //replace default inner-html values
+  fillInValues(div, data) {
+    div.getElementsByClassName('name')[0].innerHTML = data.name;
+    div.getElementsByClassName('description')[0].innerHTML = data.description;
+    div.getElementsByClassName('image')[0].setAttribute('src', data.imageURL);
+    var info = div.getElementsByClassName('info')[0];
+    info.setAttribute('href', data.infoURL);
+    info.addEventListener('click', event => {
+      event.preventDefault();
+      window.open(data.infoURL);
+    });
```

Comment:
This click handler is to open the infoURL in a new tab, right? Would it be better to do `info.setAttribute('target', '_blank')` [to make the link open in a new tab](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/a#attr-target)?